tex/artikel/Muenster_ist.tex

```diff
@@ -9,8 +9,8 @@ \section{Münster ist\dots}
 \end{center}
 
 \begin{multicols}{2}
-Münster ist eine Universitäts- und Verwaltungsstadt in Westfalen, die mit knapp \num{315000} Einwohnern Zentrum des Münsterlandes ist.
-Mit einer Fläche von \SI{303}{\km\squared} ergibt das einen Schnitt von ungefähr \num{1040} Einwohnern pro \si{\km\squared}.
+Münster ist eine Universitäts- und Verwaltungsstadt im Zentrum des Münsterlandes in Westfalen, in der über \num{316000} Menschen leben.
```

Comment:
An dieser Stelle hatte ich in vor 2 Wochen schon die Einwohnerzahl geändert, dann aber nicht mehr den ganzen Artikel vor dem Meeting geschafft. Die Änderung war aber trotzdem in einem meiner gemergten PRs, deshalb das Problem. Bitte einfach mit deiner Version überschreiben.

Reply:
Okay! Möchtest du die Gelegenheit nutzen, um die "Resolve conflicts"-Funktion unten auszuprobieren, oder soll ich das machen?

Reply:
Mach du das am besten. Ich habs mir gerade angeschaut bin aber nicht 100% sicher, dass ich alles richtig mache. Ich frag dich dann beim nächsten Meeting oder so nochmal zum erklären :D